torch_xla/csrc/convolution.cpp

```diff
@@ -33,19 +37,19 @@ namespace {
  *   - grad_input: conv(grad_output, weight^T) (with padding etc)
  *   - grad_weight: conv(input^T, grad_output)
  *
- * XLA provides the following wrappers instead of calling into raw
- * ConvGeneralDilated.
+ * PyTorch creates the following wrappers instead of calling into raw
+ * ConvGeneralDilated. And this code inspired from TF:
```

Comment:
you can remove
```
PyTorch creates the following wrappers instead of calling into raw
 * ConvGeneralDilated.
```
here. Jut say
```
Below helpers are inspired by TF2XLA implementation of the Convolution 
 *   - PTXLAMakeXlaBackpropInputConvOp
 *   - PTXLAMakeXlaBackpropFilterConvOp
```

Reply:
Also why Couldn't we just call it `MakeXlaBackpropInputConvOp`, they are in different namespace and should not conflict even if we include both.

Reply:
Thanks, Jack, updated